pyblish_nuke/nuke_path/menu.py

```diff
@@ -2,11 +2,8 @@
 
 import nuke
 
-logging.basicConfig(level=logging.INFO)
```

Comment:
That's true, I also remember something along those lines.

We should have a test for this.. but I don't know how to implement that against nuke..